components/CropSelector/CropSelector.permissions.comp.cy.js

```diff
@@ -9,21 +9,24 @@ describe('Test the permission based CropSelector content', () => {
 
   it('Admin can add crops and url is correct', () => {
     const readySpy = cy.spy().as('readySpy');
+    const addClickedSpy = cy.spy().as('addClickedSpy');
 
     cy.mount(CropSelector, {
       props: {
         includeGreenhouses: true,
         onReady: readySpy,
+        onAddClicked: addClickedSpy,
```

Comment:
The `CropSelector` should handle the `add-clicked` event emitted from the `Selector` base, clear the cache and go to the new URL.  It will not emit an `add-clicked` event of its own.  So the `addClickedSpy` here doesn't get called.  The tests for the parent components (e.g. `CropSelector`) will need to use the intercept approach that was in `SelectorBase` to be sure that they go to the correct URL when they receive an `add-clicked` event.